src/hotspot/cpu/x86/sharedRuntime_x86_64.cpp

```diff
@@ -1602,15 +1600,25 @@ nmethod* SharedRuntime::generate_native_wrapper(MacroAssembler* masm,
     } else if (method->is_continuation_yield_intrinsic()) {
       gen_continuation_yield(masm,
                              in_regs,
-                             exception_offset,
                              oop_maps,
                              frame_complete,
                              stack_slots,
-                             interpreted_entry_offset,
                              vep_offset);
     } else {
       guarantee(false, "Unknown Continuation native intrinsic");
     }
+
+    if (method->is_continuation_enter_intrinsic()) {
```

Comment:
I suggest putting these if's inside an #ifdef ASSERT.  Or if you don't mind breaking up the asserts, move them up into the if's above.

Reply:
Done the `#ifdef ASSERT` thing in new commit.